tests/mocks/github.ts

```diff
@@ -118,7 +118,9 @@ async function getUser(request: Request) {
 }
 
 const passthroughGitHub =
-	!process.env.GITHUB_CLIENT_ID.startsWith('MOCK_') && !process.env.TESTING
+	!process.env.GITHUB_CLIENT_ID.startsWith('MOCK_') &&
+	process.env.NODE_ENV !== 'test'
```

Comment:
This looks like a left-over or a typo and not described by the env schema.  It led to the tests failing since we are bypassing the msw once real creds for GitHub OAuth app are set up.

Reply:
Thanks for fixing this! Awesome to see someone actually going through and testing what is in this offering!

I posted in the epic-stack-help on 01/09/2024 and heard nothing.

https://discord.com/channels/715220730605731931/1194257050394443816/1194257050394443816
```Has anyone tried running npm run test after setting a valid GITHUB_CLIENT_ID in your .env file just to see a bunch of auth.$provider.callback.test.ts tests fail?

It seems const passthroughGitHub = !process.env.GITHUB_CLIENT_ID.startsWith('MOCK_') && !process.env.TESTING isn't quite right.

If you step through a debugging session you'll see process.env.TESTING is undefined.

Do we know if this check should be referencing a value that is actually defined such as:
process.env_MODE !== 'test' or 
process.env_NODE_ENV !== 'test' or
!process.env_TEST or
!process.env_VITEST

Thanks!```